Mathlib/Data/List/Basic.lean

```diff
@@ -329,7 +329,7 @@ theorem getLast_replicate_succ (m : ℕ) (a : α) :
 
 /-- If the last element of `l` does not satisfy `p`, then it is also the last element of
 `l.filter p`. -/
-lemma getLast_filter {p : α → Bool} :
+lemma getLast_filter' {p : α → Bool} :
```

Comment:
There's a new theorem with the same name (and completely different statement) in Lean.